### ISSUE TITLE
Components include scripts that use jQuery early in HTML body

### ISSUE THREAD

**Issue by hackel** [Nayjest/Grids#113]
It's standard practice to load all Javascript libraries like jQuery at the very end of an HTML document.  As a result, the scripts in various components that use $() return errors, because that function is not yet defined.  These scripts need to be added to a @content section which can be yielded at the bottom of the grid template, *after* loading jQuery.

Another possible, simple, non-blade solution would be something like this: http://stackoverflow.com/a/21922298/4186945 but it still requires the dev to add the two code blocks to the start/end of their template.

Also, the root `$(function () {})` should use the long-form `jQuery(function () {})` to work with jQuery.noConflict() and avoid conflicts with other libraries.

**Comment by Nayjest** [comment]
Hi,
Regarding  long-form jQuery -- no problem, it's easy to fix, if you want to get that changes faster, you can replace $ to jQuery yourself & create pull request, I will merge that & release during day.

Regarding moving jQuery to bottom:
1. I don't think that direct use of solution mentioned on stackoverflow is a good idea for grids package, becouse it will require end-user to include additional scripts like 
```html
<script type="text/javascript">
    jQuery(function(){
      while (domReadyQueue.length) {
        domReadyQueue.shift()(jQuery);
      }
    });
  </script>
```
manually, it makes usage of grids more complex & broke backward compatibility.

2. In [next major releases of grids](https://github.com/presentation-framework) it's easy to fix becouse presentation framework has resource manager class that controls including JS/CSS resources to page. With current repository I can't broke backward compatibility, so not all solutions can be accepted.

3. I suggest we need just to review all JS in package: possibly some things doesn't require wrapping to $(function () {}) at all (becouse substantially there are event handlers) and some other parts of code can be written using vanilla js without increasing it's complexity.

**Comment by the-damned** [comment]
We can just replace $(function(){...}) with window.onload = function() {...}. 
That helps to me.

**Comment by Nayjest** [comment]
Released in v1.3.5
